packages/api/src/routers/api/alerts.ts

```diff
@@ -71,6 +72,52 @@ const validateGroupBy = async (req, res, next) => {
 };
 
 // Routes
+router.get('/', isUserAuthenticated, async (req, res, next) => {
+  try {
+    const teamId = req.user?.team;
+    if (teamId == null) {
+      return res.sendStatus(403);
+    }
+    const alerts = await Alert.find({ team: teamId });
+    // may want to restrict this to reasonable time bounds
+    const alertHistories = await AlertHistory.find({ team: teamId });
```

Comment:
Any reason we want to fetch this ?

Reply:
This is used to draw the alert history on the page, i.e. the sparkline. I've limited it to 20 per alert in the last commit :)